test/browser/render.test.js

```diff
@@ -464,6 +464,26 @@ describe('render()', () => {
 			expect(style.zIndex.toString()).to.equal('');
 		});
 
+		it('should remove existing attributes', () => {
+			const div = document.createElement('div');
+			div.setAttribute('class', 'red');
+			const span = document.createElement('span');
+			const text = document.createTextNode('Hi');
+
+			span.appendChild(text);
+			div.appendChild(span);
+			scratch.appendChild(div);
+
+			const App = () => (
+				<div>
+					<span>Bye</span>
+				</div>
+			);
+
+			render(<App />, scratch);
+			expect(scratch.innerHTML).to.equal('<div class=""><span>Bye</span></div>');
```

Comment:
Huh... I guess `class` should be missing entirely here, based on this https://github.com/preactjs/preact/pull/1965. Does it makes sense? A previous attribute, removed entirely from node if that's missing. I guess it kinda does.

Reply:
className also resolves to an empty string it seems

Reply:
Yup. I guess this will surface in my PR. I'll handle it then and ping you 😛